great_expectations/cli/datasource.py

```diff
@@ -44,26 +44,9 @@
 logger = logging.getLogger(__name__)
 
 
-class DatasourceTypes(enum.Enum):
```

Comment:
Nevermind - saw that they are moved now :)

Reply:
Yes, interestingly -- when I did the diff, I saw this and did a double-take; and then saw they moved.  I do not think I did that, because I do not recall having a reason to do so (am I forgetting?!) -- so it could be something our linters did...

Reply:
Either way, it seems like they are in a better file now! 😄